### DISCUSSION TITLE
Minio console behind reverse proxy redirect

### DISCUSSION THREAD

**Discussion by 12nick12** [minio/minio#13084] [Q&A] [answered]
Hi, I have minio behind a NGiNX reverse proxy for the console and API. I was curious how do I go about telling the api -> console redirect where to redirect to? Right now when I go to the API in Edge/Firefox it tried to redirect to api_url:3000 which makes sense, but how do I override the redirect URL to console.domain.com? Thank you guys for an awesome project.

**Comment by harshavardhana** [comment, marked as answer]
https://github.com/minio/minio#things-to-consider please read here.

**Comment by 12nick12** [comment]
awesome, thank you so much. I didn't realize there was a discussions tab.
